src/capi-single-supported/index.js

```diff
@@ -14,12 +14,11 @@ if (customUrl !== '') {
   params.append('k', keywords);
 };
 
-enableToggles();
 getIframeId()
 .then(() => fetch(`https://api.nextgen.guardianapps.co.uk/commercial/api/capi-single.json?${params}`))
 .then(response => response.json())
-.then(capiData => [capiData.articleImage, populateCard(capiData)])
-.then(([imageInfo, html]) => Promise.all([getWebfonts(), write(() => container.innerHTML = html)]).then(() => addImage(imageInfo)))
+.then(capiData => [populateLogo(capiData), populateCard(capiData)])
+.then(([logo, body, imageInfo]) => Promise.all([getWebfonts(), write(() => header.innerHTML = logo, container.innerHTML = body).then(() => addImage(imageInfo))]))
```

Comment:
how come the array argument has three elements in it, if the previous callback returns an array of only two?